README.md

```diff
@@ -120,14 +120,14 @@ Projects that require either of the two features need to explicitly declare the
 <dependency>
 	<groupId>com.microsoft.sqlserver</groupId>
 	<artifactId>mssql-jdbc</artifactId>
-	<version>6.3.6.jre8-preview</version>
```

Comment:
DO you think this should be updated to jre7 since this is JRE7 branch?

Reply:
We shall keep it jre8 since this is just the same state as old dev branch used to be. We never used jre7 as we generally use latest supported versions in current branch in readme.